### DISCUSSION TITLE
x-mask should have option for $percentage also like $money

### DISCUSSION THREAD

**Discussion by aaqibmehran** [alpinejs/alpine#3741] [3. Feature ideas]
It should support most of the percentage formats. i.e (0, 2.5, 22.5, 22.55, 100) with min and max settings.

**Comment by ekwoka** [comment]
You can make the case for it, or make a plugin for it.

Just add your own magic that returns a transformer function :)

**Comment by ahmedkandel** [comment]
@aaqibmehran you can use dynamic mask as following:

```
<input x-mask:dynamic="
    parseFloat($input) > 100 ? '100' : ($input[2] === '.' ? '99.99' : ($input[1] === '.' ? '9.99' : '999'))
">
```
